packages/plans-grid-next/src/components/shared/tooltip/style.scss

```diff
@@ -0,0 +1,22 @@
+.plans-grid-next-tooltip {
+	&.tooltip.popover .popover__inner {
+		background: var(--color-masterbar-background);
```

Comment:
I think we should consider removing the restyling here altogether, or keep the bare minimum. Is there reason to deviate from Core on this?

Reply:
Let's followup with any visual changes as a separate task (https://github.com/Automattic/martech/issues/3321). Any change will need to be kept in sync with `/pricing` (which uses the same modified style).